minio/api.py

```diff
@@ -599,7 +599,12 @@ def fget_object(self, bucket_name, object_name, file_path, request_headers=None)
                 raise InvalidSizeError(msg)
 
         # Rename with destination file.
-        os.rename(file_part_path, file_path)
+        if not(os.path.exists(file_path)):
```

Comment:
This can be changed to ```if os.path.exists(file_path):
                                                os.remove(file_path)
                                          os.rename(file_part_path, file_path) ```